lib/plugins/cannulaage.js

```diff
@@ -12,40 +12,83 @@ function init() {
   cage.label = 'Cannula Age';
   cage.pluginType = 'pill-minor';
 
-  cage.updateVisualisation = function updateVisualisation (sbx) {
-    var age = 0;
-    var found = false;
-    var treatmentDate = null;
-    var message = '';
-
-    _.forEach(sbx.data.treatments, function eachTreatment (treatment) {
-      if (treatment.eventType === 'Site Change') {
-        treatmentDate = new Date(treatment.created_at);
-        var hours = Math.round(Math.abs(sbx.time - treatmentDate) / 36e5);
-
-        if (!found) {
-          found = true;
-          age = hours;
+  cage.checkNotifications = function checkNotifications(sbx) {
+  
+    // Just return if no notification times are configured
+  
+  	if (!sbx.extendedSettings.urgent && !sbx.extendedSettings.warn) return;
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Expected '{' and instead saw 'return'.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/768506884/issues/source?bid=2452379&fileBranchId=2457543#l19)